test/nocks.ts

```diff
@@ -13,16 +13,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-'use strict';
 
-var nock = require('nock');
+ var nock = require('nock');
```

Comment:
nit: unnecessary whitespace. It is okay if it is fixed in a follow-on (if fixing this requires that you have to rebase all your pending PRs).